t/proto_template/spec.yaml

```diff
@@ -17,22 +17,29 @@ kind: 'Template'
 
 desc:
   'A template that sets up a protos directory along with a script to generate language specific protos.'
+
 inputs:
   - name: 'github_org_name'
     desc: 'The name of the github org the repo will belong to'
   - name: 'github_repo_name'
     desc: 'The name of the github repo'
+  - name: 'protobuf_version'
+    desc: 'The protobuf version to use, see https://github.com/protocolbuffers/protobuf/releases'
+    default: '24.3'
+
 steps:
   - desc: 'Include some files and directories'
     action: 'include'
     params:
-      paths: ['protos', 'scripts']
+      paths: ['protos', 'scripts', '.github', 'prototool.yaml']
   - desc: 'Replace "github_org_name" and "github_repo_name" with your organization and repo name.'
     action: 'string_replace'
     params:
-      paths: ['protos/v1/person_service/person.proto', 'scripts/generate.sh']
+      paths: ['protos/v1/person_service/person.proto', 'scripts/generate.sh', '.github/workflows/ci.yml', 'prototool.yaml']
```

Comment:
If you want, you can simplify by doing `paths: ['.']` to just replace in every file that has been included.

Reply:
ty